README.md

```diff
@@ -219,5 +220,22 @@ function toggleHidden(element) {
 	}
 }
 ```
+[↑ back to Table of Contents](#table-of-contents)
+
+## Use `.first()` for single element
+
+### Why?
+jQuery always returns a collection when using `$(selector)` or `$root.find(selector)`, while sometimes you are only interested in / only expect one element. In vanilla JS you would use `.querySelector(selector)` instead of `.querySelectorAll(selector)`. For clarity you should use [`.first()`](http://api.jquery.com/first/).
+
+### How?
```

Comment:
Describe "How?" in text before code snippet?: "Use `first()` to get the first item of the collection. Also ensure variable name reflects its a single element."